cmd/main.go

```diff
@@ -45,7 +45,7 @@ import (
 	"github.com/openshift/assisted-service/pkg/app"
 	"github.com/openshift/assisted-service/pkg/auth"
 	paramctx "github.com/openshift/assisted-service/pkg/context"
-	"github.com/openshift/assisted-service/pkg/db"
+	dbPkg "github.com/openshift/assisted-service/pkg/db"
```

Comment:
Isn't the convention not to camelCase in package names?

Why change this at all?

Reply:
Since we use **a lot** `db` as a variable, I found it more suitable to alias the package.
Please, let me know if there's another naming you prefer.